spark/src/main/scala/org/apache/spark/sql/delta/DeltaOperations.scala

```diff
@@ -59,6 +60,19 @@ object DeltaOperations {
 
     /** Whether this operation changes data */
     def changesData: Boolean = false
+
+    /**
+     * Manually transform the deletion vector metrics, because they are not part of
+     * `operationMetrics` and are filtered out by the super.transformMetrics() call.
+     */
+    def transformDeletionVectorMetrics(
```

Comment:
Should we make this change when DV metrics are populated in the DELETE command?

Reply:
Yes, it's better to keep the `added + updated` logic separate from DELETE because we want metrics produced by all DML commands to go through this logic.
In the upcoming PR the UPDATE command will call this method.